src/handlers/indexers/start_indexer.rs

```diff
@@ -62,6 +65,13 @@ pub async fn start_indexer(id: Uuid) -> Result<(), IndexerError> {
     Ok(())
 }
 
+pub async fn start_indexer_api(
```

Comment:
we should check for the current indexer status

Reply:
this is already checked inside start_indexer

Reply:
right I mean we should add support for re-starting an indexer that failed running

Reply:
added!